Common/Product/ReplWindow/Repl/OutputBuffer.cs

```diff
@@ -140,17 +137,17 @@ private void AppendEscapedText(string text, bool isError, int escape)
                             if (codes.Count == 0)
                             {
                                 // reset
-                                color = isError ? ConsoleColor.Red : ConsoleColor.White;
+                                color = isError ? InteractiveWindowColor.Red : InteractiveWindowColor.White;
```

Comment:
Above you change _errColor from Red to `_errColor = InteractiveWindowColor.Error`. Why hard code to Red again here?